src/edit_collector.h

```diff
@@ -121,20 +128,33 @@ class EditCollector {
  private:
   class CFEditCollector {
    public:
+    CFEditCollector(Logger* logger, bool _paranoid_check)
+        : paranoid_check_(_paranoid_check), info_log_(logger) {}
+
     Status AddFile(const std::shared_ptr<BlobFileMeta>& file) {
       auto number = file->file_number();
       if (added_files_.count(number) > 0) {
-        return Status::Corruption("Blob file " + ToString(number) +
-                                  " has been added twice");
+        TITAN_LOG_ERROR(info_log_,
```

Comment:
TITAN_LOG_ERROR can handle info_log_  is null right?

Reply:
yes